src/Tribe/REST/V1/Endpoints/Attendee_Archive.php

```diff
@@ -108,7 +108,7 @@ public function get( WP_REST_Request $request ) {
 			'status'                         => 'post_status',
 			'order_status'                   => 'order_status',
 			'checkedin'                      => 'checkedin',
-			'rsvp_going_status'              => 'rsvp_status',
+			'rsvp_going_status'              => 'rsvp_status__or_none',
```

Comment:
Just curios here on the reason on the amount of `_` for `rsvp_status__or_none` as it does not seems to follow a pattern, asking if is intentional?

Reply:
This change is to ensure backwards compatibility with previous REST API requests that included the RSVP status as an optional / match value. The normal pattern for filtering exact matches wasn't followed originally so this resolves that.

Reply:
Oh! My question was more about the usage of double underscore to separate one word and one for another. 

`rsvp_status` `__` `or_none`

Reply:
Ah, yes the double underscore is intentional, although I've pinged Gustavo/Luca about this because I misinterpreted our position on following the WordPress usage.

Reply:
I'm going to say we keep this since ET has already established this pattern for itself https://github.com/moderntribe/event-tickets/blob/master/src/Tribe/Attendee_Repository.php#L45